UPGRADE.md

```diff
@@ -119,7 +123,7 @@ services:
 <details>
   <summary>Sulu Google Cloud Config to Flysystem S3</summary>
 
-> [!NOTE]  
```

Comment:
Please avoid unrelated moficationsin the UPGRADE.md file:

PS: space has special meaning in md files.
If the IDE supports it we configure in the editorconfig file to avoid this:

```
[*.md]
trim_trailing_whitespace = false
```

But still you can in most IDEs manually configure this per file extension via the UI.